discovery/gce/gce.go

```diff
@@ -178,6 +178,15 @@ func (d *Discovery) refresh(ctx context.Context) ([]*targetgroup.Group, error) {
 			addr := fmt.Sprintf("%s:%d", priIface.NetworkIP, d.port)
 			labels[model.AddressLabel] = model.LabelValue(addr)
 
+			// Append additional interface private IP address metadata if multiple interfaces exist.
+			if len(inst.NetworkInterfaces) > 1 {
```

Comment:
There's no need to make this conditional, we should always include the extra metadata. This avoids problems with regexp matching where you want to default to one, but if a second exists, override.

Reply:
ack.  updated.